javascript/src/test/components/OfficeHours/OfficeHourTable.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render, waitForElement, waitFor } from "@testing-library/react";
+import OfficeHourTable from "main/components/OfficeHours/OfficeHourTable";
+import userEvent from "@testing-library/user-event";
+
+
+describe("OfficeHour table tests", () => {
+
+  const sampleOfficeHour = [{
+    "id": 1,
+    "name": "CMPSC 156",
+    "quarter": "F20",
+    "instructorFirstName": "Phill",
+    "instructorLastName": "Conrad",
+    "instructorEmail": "phtcon@ucsb.edu"
+
+  }]
+
+
+
+  test("renders without crashing", () => {
+    render(<OfficeHourTable officeHours = {sampleOfficeHour}/>);
+  });
+
+/*  test("Test delete button calls func", () => {
```

Comment:
Why is this test commented out?   If the delete function hasn't been written yet, then we can delete this test.  If it has been written and isn't being tested, we should test it (But then you'd probably get coverage problems, right?)

Reply:
I believe test got moved to a more applicable location for testing.